src/commissaire_service/investigator/__init__.py

```diff
@@ -170,7 +170,6 @@ def on_investigate(self, message, address, cluster_data={}):
                     network.name, network.type))
             transport.bootstrap(
                 address, key.path, oscmd, etcd_config, network)
-            host.status = 'inactive'
```

Comment:
Looking at the code the last request looks like it's just removing the ``disassociated``. The logic looks the same as above.

:arrow_up: